docs/bigquery-usage.rst

```diff
@@ -0,0 +1,49 @@
+Using the API
+=============
+
+Authorization / Configuration
+-----------------------------
+
+- Use :class:`Client <gcloud.bigquery.client.Client>` objects to configure
+  your applications.
```

Comment:
I'm hoping to actually write auth docs once #952 is resolved. (Recall both JJ and I did this before, but the PRs got left behind.)